css/nixos-site.css

```diff
@@ -183,6 +189,10 @@ pre {
     content: " [nix-shell]$ ";
 }
 
+div.main.full-width {
+  max-width: 40em;
+}
```

Comment:
The `full-width` name is counter-intuitive. I would have assumed, in a non-semantic way, that this means the element will use the full width of the parent container, rather than be constrained.